docs/annexes/spdx-license-expressions.md

```diff
@@ -46,48 +46,48 @@ A valid `<license-expression>` string consists of either:
 
 (ii) a more complex expression constructed by combining smaller valid expressions using Boolean license operators.
 
-There MUST NOT be white space between a license-id and any following `+`. This supports easy parsing and backwards compatibility. There MUST be white space on either side of the operator "WITH". There MUST be white space and/or parentheses on either side of the operators `AND` and `OR`.
+There MUST NOT be any space between a license-id and any following `+`. This supports easy parsing and backwards compatibility.
 
-In the `tag:value` format, a license expression MUST be on a single line, and MUST NOT include a line break in the middle of the expression.
+There MUST be at least one space on either side of the operators `AND`, `OR`, and "WITH".
+
+A license expression MUST be on a single line, and MUST NOT include a line break in the middle of the expression.
 
 ## Case sensitivity
 
-License expression operators (`AND`, `and`, `OR`, `or`, `WITH` and `with`) should be matched in a *case-sensitive* manner, i.e., letters must be all upper case or all lower case.
+In SPDX-3, license expressions are completely *case-insensitive*.
 
-License identifiers (including license exception identifiers) used in SPDX documents or source code files should be matched in a *case-insensitive* manner. In other words, `MIT`, `Mit` and `mIt` should all be treated as the same identifier and referring to the same license.
+That includes the operators (`AND`, `OR`, `WITH`), the special identifiers (`NONE` and `NOASSERTION`), as well as the license identifiers, including the user-defined ones.
```

Comment:
```suggestion
That includes the operators (`AND`, `OR`, `WITH`), the special identifiers ("NONE" and "NOASSERTION"), as well as the license identifiers, including the user-defined ones.
```

NONE and NOASSERTION are found more with double quotes elsewhere in this document.